packages/react-components/src/components/CaptionsSettingsModal.tsx

```diff
@@ -90,15 +90,20 @@ export const _CaptionsSettingsModal = (props: _CaptionsSettingsModalProps): JSX.
     }
   }, [onDismissCaptionsSettings]);
 
+  useEffect(() => {
```

Comment:
My idea is since spoken language doesn't change, it's ok if setspokenlanguage is called multiple times. Calling setspokenlanguage(english) twice will not change the spoken language

Reply:
That is true for 1 customer scenario, but imagine if you have 2 more customers, it could be an easy raise condition to trigger this regression

Reply:
Since we always require the modal to set language when starting captions, we don't have a scenario where selectedItem.key is different across participants?